crates/storage-pg/src/queue/schedule.rs

```diff
@@ -69,7 +69,7 @@ impl QueueScheduleRepository for PgQueueScheduleRepository<'_> {
             ScheduleLookup,
             r#"
                 SELECT
-                    queue_schedules.schedule_name,
+                    queue_schedules.schedule_name as "schedule_name!",
```

Comment:
Not sure why, but my local sqlx kept complaining about this, even though it's supposed to be NOT NULL already ;(